styx-scheduler-service/src/test/java/com/spotify/styx/docker/KubernetesDockerRunnerPodPollerTest.java

```diff
@@ -22,11 +22,11 @@
 
```

Comment:
I think we can merge this file to `KubernetesDockerRunnerTest.java` since there is no `poller` anymore. Or rename this file.

Reply:
Sure, let's do that in another PR.